backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxRuleApi.scala

```diff
@@ -38,18 +38,18 @@ class VeloxRuleApi extends RuleApi {
   override def injectRules(injector: RuleInjector): Unit = {
     injector.gluten.skipOn(PhysicalPlanSelector.skipCond)
 
-    injectSpark(injector.spark)
+    injectSpark(injector)
     injectLegacy(injector.gluten.legacy)
     injectRas(injector.gluten.ras)
   }
 }
 
 private object VeloxRuleApi {
-  def injectSpark(injector: SparkInjector): Unit = {
-    // Regular Spark rules.
-    injector.injectOptimizerRule(CollectRewriteRule.apply)
-    injector.injectOptimizerRule(HLLRewriteRule.apply)
-    injector.injectPostHocResolutionRule(ArrowConvertorRule.apply)
+  def injectSpark(injector: RuleInjector): Unit = {
```

Comment:
Can we make it 

```suggestion
  def injectSpark(exts: SparkSessionExtensions): Unit = {
```

? Thanks.

Reply:
How about `extensions`?